browser/extensions/api/binance_api.cc

```diff
@@ -261,21 +261,25 @@ BinanceGetConvertAssetsFunction::Run() {
 }
 
 void BinanceGetConvertAssetsFunction::OnGetConvertAssets(
-    const std::map<std::string, std::vector<std::string>>& assets) {
-  std::unique_ptr<base::DictionaryValue> asset_dict(
-      new base::DictionaryValue());
+    const std::map<std::string, std::vector<
+    std::map<std::string, std::string>>>& assets) {
```

Comment:
Totally up to you, but a typedef might help here to ease readability.

Reply:
Agreed. I plan to do this in a follow up PR as there quite a few examples of responses like this throughout the widget API’s, so I can get them all in one go